handouts/git-course-handout.tex

```diff
@@ -90,7 +90,7 @@
   \item \textbf{git add} -- See ``git stage''
```

Comment:
Should this be set to have `git stage` as default instead of `git add`? (And make `git add` as optional?)

Reply:
I think this is what I've done... right?

Reply:
Phew! I just saw the line above. Sorry, yes you are right!

Reply:
Maybe instead of just saying "see `git stage`" we should include some detail like "Old name for `git stage`. Kept for backwards-compatibility"

Reply:
Technically it's not the old name -- `git add` isn't deprecated or anything. How about we put "equivalent to `git stage`" or something?